packages/config-schema/src/schema/netlify-toml.schema.json

```
@@ -0,0 +1,659 @@
+        },
+        "links": {
+          "key": "https://docs.netlify.com/routing/redirects/rewrites-proxies/#custom-headers-in-proxy-redirects"
+        }
+      }
+    },
+    "functions": {
+      "title": "Functions",
+      "description": "Although there are default settings for Netlify Functions to help you get started, you can use this section for optional, custom configuration.",
+      "type": "object",
+      "additionalProperties": {
+        "description": "Set these options for specific functions by filtering them by function name. You can use glob patterns in your filter. If a function matches several configuration blocks containing one of these properties, the values are concatenated.",
+        "type": "object",
+        "properties": {
+          "node_bundler": { "$ref": "#/definitions/FunctionConfigNodeBundler" },
+          "external_node_modules": { "$ref": "#/definitions/FunctionConfigExternalNodeModules" },
+          "included_files": { "$ref": "#/definitions/FunctionConfigIncludedFiles" }
+        },
+        "additionalProperties": false
+      },
+      "properties": {
+        "deno_import_map": {
+          "title": "Import maps",
+          "description": "Relative path to an import map file to map module URLs to names.",
+          "type": "string"
+        },
+        "external_node_modules": { "$ref": "#/definitions/FunctionConfigExternalNodeModules" },
+        "included_files": { "$ref": "#/definitions/FunctionConfigIncludedFiles" },
+        "node_bundler": { "$ref": "#/definitions/FunctionConfigNodeBundler" },
+        "directory": {
+          "title": "Directory",
+          "description": "Custom path to your functions. The default location is `YOUR_BASE_DIRECTORY/netlify/functions`.",
+          "type": "string",
+          "default": "netlify/functions"
+        }
+      },
+      "x-taplo": {
+        "links": {
+          "key": "https://docs.netlify.com/configure-builds/file-based-configuration/#functions"
+        }
+      }
+    },
+    "edge_functions": {
+      "type": "array",
+      "default": [{ "path": "/path", "function": "function-name" }],
+      "items": {
+        "title": "Edge Functions",
+        "description": "While you can declare edge functions inline, you can use this section for more advanced edge function configurations.",
+        "type": "object",
+        "properties": {
+          "cache": {
+            "type": "string",
+            "description": "Should the response of the edge function be cached.",
+            "enum": ["off", "manual"],
+            "x-taplo": {
+              "docs": {
+                "enumValues": [
+                  "Default, edge function runs without caching.",
+                  "Edge function response will be cached in the Netlify CDN depending on returned caching headers."
```

Slight edit here, again just based on the docs PR 🤔 

```suggestion
                  "Edge function responses are cached in the Netlify CDN based on the specified `cache-control` headers."
```

---

looking at this made me realize that [some of the language in the docs PR is overly specific](https://github.com/netlify/docs/pull/2928/files#r1153823487). So, I'm going to suggest a mash-up of the original and @codebyuma 's suggestion

```suggestion
                  "Edge function response is cached in the Netlify CDN based on the specified caching headers."
```